NGitLab/Models/PackageSort.cs

```diff
@@ -0,0 +1,8 @@
+namespace NGitLab.Models
+{
+    public enum PackageSort
+    {
+        asc,
```

Comment:
Use C# name (e.g. `Default`) and the `EnumMember(Value = "")]` attribute to map the actual name